docs/development.md

```diff
@@ -0,0 +1,35 @@
+# Development
+
+This document should entail all you need to develop this operator locally. 
+
+## Versions
+
+### golang
+
+`go version go1.13`
+
+### operator-sdk
+
+`operator-sdk version: "v0.17.0", commit: "2fd7019f856cdb6f6618e2c3c80d15c3c79d1b6c", kubernetes version: "v1.17.2", go version: "go1.14.2 linux/amd64"`
```

Comment:
Golang version above mentions of 1.13 and the client version in operator-sdk version shows go1.14.2. We could mark it same to avoid confusion?

Reply:
good catch